pages/common/atool.md

```diff
@@ -0,0 +1,37 @@
+# atool
+
+> A script for managing file archives of various types.
+> `atool` uses external archiver programs but provides a consistent command-line interface for listing, extracting, creating, and managing archives.
+> More information: <https://savannah.nongnu.org/projects/atool/>.
```

Comment:
```suggestion
> More information: <https://manned.org/atool>.
```
Unless you can prove me otherwise, this link does not provide any meaningful documentation.